src/Lykke.Exchange.Api.MarketData/PeriodicalHandlers/CleanupHandler.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Autofac;
+using Common;
+using JetBrains.Annotations;
+using Lykke.Common.Log;
+using Lykke.Exchange.Api.MarketData.Services;
+using Lykke.Service.MarketProfile.Client;
+using StackExchange.Redis;
+
+namespace Lykke.Exchange.Api.MarketData.PeriodicalHandlers
+{
+    [UsedImplicitly]
+    public class CleanupHandler : IStartable, IDisposable
+    {
+        private readonly ILykkeMarketProfile _marketProfileClient;
+        private readonly IDatabase _database;
+        private readonly TimerTrigger _timerTrigger;
+
+        public CleanupHandler(
+            ILykkeMarketProfile marketProfileClient,
+            IDatabase database,
+            ILogFactory logFactory
+            )
+        {
+            _marketProfileClient = marketProfileClient;
+            _database = database;
+            _timerTrigger = new TimerTrigger(nameof(CleanupHandler), TimeSpan.FromMinutes(10), logFactory);
+            _timerTrigger.Triggered += Execute;
+        }
+
+        public void Start()
+        {
+            _timerTrigger.Start();
+        }
+
+        public void Dispose()
+        {
+            _timerTrigger?.Stop();
+            _timerTrigger?.Dispose();
+        }
+
+        private async Task Execute(ITimerTrigger timer, TimerTriggeredHandlerArgs args, CancellationToken cancellationToken)
+        {
+            double from = DateTime.UtcNow.AddHours(-24).ToUnixTime();
+            var assetPairIds = (await _marketProfileClient.ApiMarketProfileGetAsync(cancellationToken))
+                .Select(x => x.AssetPair)
+                .ToList();
+
+            var tasks = new List<Task>();
+
+            foreach (var assetPairId in assetPairIds)
```

Comment:
I'm not sure if it's necessary here, but in Redis we can use the Transaction primitive to execute all actions by creating tranasaction and waiting for a single task. The same applies for all other group operations in this PR.